storybook/tailwind.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  purge: ["./src/*.html"],
+  purge: ["./src/**/*.{js,jsx,ts,tsx}", "./public/index.html"],
```

Comment:
good catch. 

One more tip, note that using dynamic class names in the component, so always avoid having something like the following. 

```
<div className={'text-'+size} />
```

the purge topic is kinda tricky and I am happy to see you already touch it. good job!

Reply:
good catch. 

One more tip, note that using dynamic class names in the component will limit the automatic purging, so always avoid having something like the following. 

```
<div className={'text-'+size} />
```

the purge topic is kinda tricky and I am happy to see you already touch it. good job!